src/pocketdb/consensus/social/BoostContent.hpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2018-2021 Pocketnet developers
+// Distributed under the Apache 2.0 software license, see the accompanying
+// https://www.apache.org/licenses/LICENSE-2.0
+
+#ifndef POCKETCONSENSUS_BOOSTCONTENT_HPP
+#define POCKETCONSENSUS_BOOSTCONTENT_HPP
+
+#include "pocketdb/consensus/Social.h"
+#include "pocketdb/models/dto/BoostContent.h"
+
+namespace PocketConsensus
+{
+    using namespace std;
+    typedef shared_ptr<BoostContent> BoostContentRef;
+
+    /*******************************************************************************************************************
+    *  BoostContent consensus base class
+    *******************************************************************************************************************/
+    class BoostContentConsensus : public SocialConsensus<BoostContent>
+    {
+    public:
+        BoostContentConsensus(int height) : SocialConsensus<BoostContent>(height) {}
+
+        ConsensusValidateResult Validate(const CTransactionRef& tx, const BoostContentRef& ptx, const PocketBlockRef& block) override
```

Comment:
Since we are adding these from new I would like to see unit tests added which verify correct boost transaction and invalid boost transaction with edge cases tested.

Reply:
Could you add test cases for this new transaction?